styles.css

```diff
@@ -20,6 +20,8 @@ a {
 
 .frontpage section {
   background-image: url(./Images/background/header-shapes\ mobile@2x.svg);
+  background-repeat: no-repeat;
+  background-size: 100%;
   margin: 0 3% 0 3%;
   padding-top: 15%;
```

Comment:
- [ ] Please consider minimizing the spacing between the headline section and the work section and stick to the Figma design as much as possible, try reducing it to half of the current `margin-bottom` in the class `frontpage`👌🏼 

| Your Design      | Figma Design |
| ----------- | ----------- |
| ![image](https://user-images.githubusercontent.com/68030297/230243277-0f1f528b-f8f0-443e-9355-0f0dce0680d9.png)  |  ![image](https://user-images.githubusercontent.com/68030297/230243334-e5654471-0bf9-4d6e-a426-eaec7cba2983.png)  |